modules/lib/lib-content/src/test/resources/test/QueryContentHandlerTest_dsl_sort.js

```diff
@@ -0,0 +1,75 @@
+var assert = require('/lib/xp/testing.js');
+var content = require('/lib/xp/content.js');
+
+exports.sortMultiple = function () {
+
+    var result = content.query({
+            'start': 0,
+            'count': 100,
+            'sort': [
+                {
+                    'field': '_modifiedTime',
+                    'direction': 'DESC'
+                },
+                {
+                    'type': 'geodistance',
+                    'field': 'myGeoPoint',
+                    'location': {
+                        'lat': '2.2',
+                        'lon': '3.3'
+                    }
+                }
+            ],
+            'query': 'type = \'article\' AND fulltext(\'myField\', \'searching for cheese\', \'AND\') ',
+        }
+    );
+};
+
+exports.sortSingle = function () {
+
+    var result = content.query({
+            'start': 0,
+            'count': 100,
+            'sort':
+                {
+                    'field': 'displayName',
+                    'direction': 'ASC'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8706041)
```suggestion
                    "direction": 'ASC'
```